Can't build /ports/linux/dlmstp_linux.h on Alpine Linux

```
In file included from /src/ports/linux/rs485.c:73:
/src/ports/linux/dlmstp_linux.h:29:10: fatal error: bits/pthreadtypes.h: No such file or directory
   29 | #include "bits/pthreadtypes.h"
      |          ^~~~~~~~~~~~~~~~~~~~~
compilation terminated.
cc: error: /src/ports/linux/rs485.o: No such file or directory
size: 'mstpcap': No such file
cp: can't stat 'mstpcap': No such file or directory
```

I think all that needs to be done is to replace `bits/pthreadtypes.h` with `sys/types.h`.

--- a/ports/linux/dlmstp_linux.h
+++ b/ports/linux/dlmstp_linux.h
@@ -26,7 +26,7 @@
 
 #include "bacnet/datalink/mstp.h"
 /*#include "bacnet/datalink/dlmstp.h" */
-#include "bits/pthreadtypes.h"
+#include <sys/types.h>
 #include <semaphore.h>
 
 #include <stdbool.h>

